gallery.markdown

```diff
@@ -55,7 +55,7 @@ order: 1
   </script>
 </figure>
 <figure itemscope="" itemtype="https://schema.org/Painting">
-  <img src="https://github.com/naomimakkelie-nl/www2.naomimakkelie.nl/assets/62353999/47ccca17-7c11-4788-879b-e90da6921c5e" alt="Battlefield on Valentine’s Day" itemprop="image">
+  <img src="https://www2.naomimakkelie.nl/assets/img/valentines-battlefield.jpg" alt="Battlefield on Valentine’s Day" itemprop="image">
```

Comment:
Sweep make sure the valentines-battlefield.jpg file is added to the assets/img directory of the repository

Reply:
Sweep please add the image attached to the issue description (first message) to the repository in the assets/img directory